web/models/user.ex

```diff
@@ -19,7 +19,7 @@ defmodule Pairmotron.User do
   @minimum_password_length 8
 
   @required_params ~w(name email)
-  @optional_params ~w(password password_confirmation active)
+  @optional_params ~w(password password_confirmation active role_id)
```

Comment:
Not actually used in this PR, but useful for development so that you can assign roles from iex.